frontend/src/components/Forms/MineRecordForm.js

```diff
@@ -410,10 +448,12 @@ export class MineRecordForm extends Component {
 
 MineRecordForm.propTypes = propTypes;
 MineRecordForm.defaultProps = defaultProps;
+const selector = formValueSelector(FORM.MINE_RECORD);
 
```

Comment:
We're now accessing form values in two different ways, either use `formValueSelector` or `getFormValues`, I think `formValueSelector` is actually better as it returns only the value we want and not the entire object.